### PR TITLE
[ut] some operators and language cases

### PR DESCRIPTION
**Pull request by ESI-SYD**

### CODE REVIEW

File: .github/scripts/case_update.sh [hunk 1]
```diff
@@ -1,35 +1,45 @@
...
+sed -i '/capability = torch.cuda.get_device_capability/,+2 s/^/#/' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_cross_entropy.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_flash_attention.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_inductor.py
+sed -i 's/cuda/xpu/g' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_inductor.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_matmul.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/runtime/test_autotuner.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/runtime/test_cache.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/runtime/test_driver.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/runtime/test_launch.py
```
Comment (LiyangLingIntel) [inline .github/scripts/case_update.sh:42]: Could you please try to clean the duplicated commands, like
```shell
for file in ${TRITON_SRC_ROOT_DIR}/python/test/unit/runtime/*.py; do
  sed -i '/import torch/ a\import intel_extension_for_pytorch' "$file"
done
```

File: .github/scripts/case_update.sh [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
+python ${SCRIPTS_ROOT_DIR}/case_update.py ${TRITON_SRC_ROOT_DIR}/python/test/unit/language/test_core.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_blocksparse.py
+sed -i "s/device='cuda'/device='xpu'/g" ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_blocksparse.py
+sed -i '''s/device="cuda"/device="xpu"/g''' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_blocksparse.py
+sed -i '/capability = torch.cuda.get_device_capability/,+2 s/^/#/' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_blocksparse.py
+sed -i "s/cuda()/xpu()/g" ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_blocksparse.py
+sed -i '/import torch/ a\import intel_extension_for_pytorch' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_cross_entropy.py
+sed -i "s/device='cuda'/device='xpu'/g" ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_cross_entropy.py
+sed -i '''s/device="cuda"/device="xpu"/g''' ${TRITON_SRC_ROOT_DIR}/python/test/unit/operators/test_cross_entropy.py
```
Comment (LiyangLingIntel) [inline .github/scripts/case_update.sh:33]: Please make the quote `''' '''`, `' '`, `" "`  in same style, like `' '` for outer, `" "` for inner.

**Review by LiyangLingIntel — COMMENTED** [review]
Please consider group those commands for the same change together, it would be helpful for following maintain.

**Comment by ESI-SYD** [comment]
Refined

**Review by LiyangLingIntel — APPROVED** [review]
LGTM

**Pull request merged**